pkgs/by-name/fo/forgejo-runner/package.nix

```
@@ -64,21 +68,27 @@ buildGoModule rec {
   ldflags = [
     "-s"
     "-w"
-    "-X code.forgejo.org/forgejo/runner/v11/internal/pkg/ver.version=${src.rev}"
+    "-X code.forgejo.org/forgejo/runner/v12/internal/pkg/ver.version=${src.rev}"
   ];
 
   checkFlags = [
     "-skip ${lib.concatStringsSep "|" disabledTests}"
   ];
 
   postInstall = ''
-    # fix up go-specific executable naming derived from package name, upstream
+    # Fix up go-specific executable naming derived from package name, upstream
     # also calls it `forgejo-runner`
     mv $out/bin/runner $out/bin/forgejo-runner
+
+    # provide backward compatbility since v12 removed bundled git
+    wrapProgram $out/bin/forgejo-runner --prefix PATH : ${lib.makeBinPath [ git ]}
```

Not strictly an issue, but I would like to point out, that `--prefix` shadows whatever other `git` may already be in the `$PATH`.

For example, `nixos/gitea-actions-runner` adds `gitMinimal` to the systemd unit `$PATH` by default when using `:host`, but our wrapped `git` will take precedence over that `gitMinimal`.

I feel like `--suffix` might be better here, especially in the spirit of backwards compatibility. But I can also see how this discussion could go both ways.

```suggestion
    wrapProgram $out/bin/forgejo-runner --suffix PATH : ${lib.makeBinPath [ git ]}
```

Not blocking.